lib/addons/with-web/index.js

```diff
@@ -9,7 +9,6 @@ module.exports = (config = {}) => {
     } = config;
 
     config.testPathIgnorePatterns = [
```

Comment:
We no longer need declare defaults for the destructing above as they now have default values:

```js
const {
        transform = {},
        testPathIgnorePatterns = [],
        collectCoverageFrom = [],
        coveragePathIgnorePatterns = [],
    } = config;
```